packages/ssx-core/src/client/index.ts

```diff
@@ -1 +1 @@
-export * from './types';
\ No newline at end of file
+export * from './types';
```

Comment:
This goes against POSIX, lines must end in a new line.

Reply:
Checked on this and they do! Github, just renders them as a single line, if they don't have a newline the red minus, like this did

<img width="250" alt="Screenshot 2022-12-07 at 2 30 00 PM" src="https://user-images.githubusercontent.com/5846858/206277625-91976f27-a99d-48cc-abd4-28e23847cb9b.png">

Reply:
Will resolve mentions of this issue!